pkg/yurtctl/util/edgenode/util_test.go

```diff
@@ -0,0 +1,40 @@
+package edgenode
```

Comment:
please add license header.

Reply:
done

Reply:
do we have a hack script to add automatically？
and add a ci for this?

Reply:
> do we have a hack script to add automatically？
> and add a ci for this?

hack script is not provided now. and i will make an issue about it.